boot/zephyr/include/mcuboot_config/mcuboot_config.h

```diff
@@ -361,11 +361,32 @@
 #include <zephyr/device.h>
 #include <zephyr/drivers/watchdog.h>
 
+#if CONFIG_MCUBOOT_WATCHDOG_TIMEOUT
+#define MCUBOOT_WATCHDOG_INSTALL_TIMEOUT()                    \
+    do {                                                      \
+        const struct device* wdt =                            \
+            DEVICE_DT_GET(DT_ALIAS(watchdog0));               \
+        if (device_is_ready(wdt)) {                           \
+            struct wdt_timeout_cfg wdtConfig = {              \
+                .flags = WDT_FLAG_RESET_SOC,                  \
+                .window.min = 0,                              \
+                .window.max = CONFIG_MCUBOOT_WATCHDOG_TIMEOUT \
+            };                                                \
+            wdt_install_timeout(wdt, &wdtConfig);             \
```

Comment:
I believe this should be enough, as `device_is_ready(wdt)` has already been checked:
```
            struct wdt_timeout_cfg wdtConfig = {              \
                .flags = WDT_FLAG_RESET_SOC,                  \
                .window.min = 0,                              \
                .window.max = CONFIG_MCUBOOT_WATCHDOG_TIMEOUT \
            };                                                \
            wdt_install_timeout(wdt, &wdtConfig);             \
```